src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsCoreClrAttribute.cs

```diff
@@ -17,7 +17,7 @@ public bool IsMet
         {
             get
             {
-                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("coreclr");
+                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("dotnet");
```

Comment:
You could use `RuntimeInformation.FrameworkDescription`. I don't think looking for the process name is the correct check (for instance standalone apps don't have this name).